tests/parm/rrfs_conus13km_hrrr.nml.IN

```diff
@@ -0,0 +1,342 @@
+&amip_interp_nml
+    data_set = 'reynolds_oi'
+    date_out_of_range = 'climo'
+    interp_oi_sst = .true.
+    no_anom_sst = .false.
+    use_ncep_ice = .false.
+    use_ncep_sst = .true.
+/
+
```

Comment:
`amip_interp_nml` is not used.

Reply:
I will confirm this in my next round of tests.

Reply:
Confirmed. I removed that namelist and the results are identical.